sql/02-create-table-friend.sql

```diff
@@ -0,0 +1,5 @@
+DROP TABLE IF EXISTS friend;
+CREATE TABLE friend (
+	id1 INTEGER REFERENCES student(id),
+	id2 INTEGER REFERENCES student(id),
```

Comment:
A clearer naming structure for a join table would be something like:
```
student_friends (includes a reference to the student table in the table name)
   id (the primary key of the student)
   friend_id (the primary key of another student)
```
It might seem a little redundant to differentiate between the two student ids, but it would allow you to select all of a student's friends using something like:
```SQL
SELECT friend_id FROM student_friends WHERE id = (studentId)
```